internal/document/editor/editor_test.go

```diff
@@ -33,48 +32,11 @@ func TestEditor(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(
 		t,
-		document.InjectFrontmatter(string(testDataNestedFlattened)),
+		string(testDataNestedFlattened),
 		string(result),
 	)
 }
 
-func TestEditorEmptyRunme(t *testing.T) {
```

Comment:
Invalid tests related to IDT are now located in the Serialize GRPC Land.(Serialize)